src/nls/root/strings.js

```diff
@@ -584,11 +584,9 @@ define(function (require, exports, module) {
                 RECENT_DOC: "Recent Documents"
             },
             CATEGORIES: {
-                CURRENT_DOC: "document",
-                RECENT_DOC: "document",
+                CURRENT_DOC: "current document",
```

Comment:
They should be localized because they are visible as titles for the filter options and are compared to user input for autofilling and things. But yes, capitalization sounds great!